packages/explorer-2.0/pages/voting/[poll].tsx

```diff
@@ -440,14 +438,10 @@ const Poll = () => {
 }
 
 async function transformData({ poll }) {
-  let noVoteStake = parseFloat(
-    Utils.fromWei(poll?.tally?.no ? poll?.tally?.no : '0'),
-  )
-  let yesVoteStake = parseFloat(
-    Utils.fromWei(poll?.tally?.yes ? poll?.tally?.yes : '0'),
-  )
-  let totalVoteStake = parseFloat(Utils.fromWei(poll.totalVoteStake))
-  let totalNonVoteStake = parseFloat(Utils.fromWei(poll.totalNonVoteStake))
+  let noVoteStake = parseFloat(poll?.tally?.no ? poll?.tally?.no : '0')
+  let yesVoteStake = parseFloat(poll?.tally?.yes ? poll?.tally?.yes : '0')
+  let totalVoteStake = parseFloat(poll.totalVoteStake)
+  let totalNonVoteStake = parseFloat(poll.totalNonVoteStake)
```

Comment:
```suggestion
  let totalNonVoteStake = +poll?.totalNonVoteStake
```

Instead of using the null propagation operator on each of the values here we could also simply add a check at the beginning of the function; this will save us some operations. 

```
if (!poll) return;
```